deploy/charts/rook-ceph/templates/resources.yaml

```
@@ -89,6 +89,27 @@ spec:
         - jsonPath: .status.phase
           name: Phase
           type: string
+        - jsonPath: .status.info.type
+          name: Type
+          type: string
+        - jsonPath: .status.info.failureDomain
+          name: Failure Domain
+          type: string
+        - jsonPath: .spec.replicated.size
+          name: Replication Size
+          priority: 1
+          type: integer
+        - jsonPath: .spec.erasureCoded.codingChunks
+          name: Erasure Coding
```

In output for common types such as pods, deployments, replicasets, secrets, and configmaps, I see all the columns are either a single word or hyphenated such as "UP-TO-DATE" for deployments. This is more parseable for anyone who is using the output for scripting. We could use dashes or just remove spaces, or some combination. How about this?
```suggestion
          name: EC-CodingChunks
```

---

Thanks this is a very good point, I will change the naming!